### PR TITLE
Add documentation for running the analytics devstack

### PR DESCRIPTION
**Pull request by mulby**
@lamagnifica Here it is in the edx repo.

### CODE REVIEW

**Comment by lamagnifica** [comment]
@mulby please sanity check.

**Comment by mulby** [comment]
@lamagnifica Generally looks good, I noticed some outdated descriptions in here so I updated them. Can you check my changes? See the most recent commit.

**Comment by lamagnifica** [comment]
@mulby a couple of tweaks to the wording for your consideration, in the commit. Otherwise I'd say this is good to go! are we waiting on a software release or other prereq, or can this be published immediately?

**Comment by lamagnifica** [comment]
@mulby, should we add a brief mention of the Open edX Analytics Devstack in the install_options. rst file? looks like it would fit in there...
http://edx.readthedocs.org/projects/edx-installing-configuring-and-running/en/latest/install_options.html

**Comment by sarina** [comment]
Now that the vagrant file is merged, can we get this in and a final documentation link? We'd like to publish the blog post, including a link to the documentation, tomorrow afternoon or Thursday morning if possible.

**Comment by mulby** [comment]
@lamagnifica - I just added a comment to the install options doc. I'm not sure how to articulate the fact that you can develop everything on the analytics stack if you want, it's just "heavy weight". If you are aren't going to develop analytics related patches, then using the original devstack is fine, however, if you are targeting changes at insights, you should really be using the analytics devstack.

**Comment by mulby** [comment]
@sarina - agreed, this is the last piece we need to publish the blog post. I would like to have someone other than me attempt to install the devstack, though, before we publish the post to make sure it works on a bare-metal machine.

**Comment by sarina** [comment]
ah, I definitely don't have time to do that right now, unfortunately. Is
there anyone we could ask?

**Comment by lamagnifica** [comment]
@sarina I believe that this is on @dan-f 's plate

File: en_us/install_operations/source/devstack/analytics_devstack.rst [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+   
+   .. code-block:: bash
+     
+     $ paver devstack lms
+
+===================================
+Run the Open edX Analytics Data API
+===================================
+
+#. Log in to the base Open edX Devstack.
```
Comment (dan-f) [inline en_us/install_operations/source/devstack/analytics_devstack.rst]: Should say "Log in to the Analytics Devstack."

File: en_us/install_operations/source/devstack/analytics_devstack.rst [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+   
+   .. code-block:: bash
+
+     $ ~/venvs/analytics_api/bin/python ~/analytics_api/manage.py runserver 0.0.0.0:8100 --insecure
+
+=====================
+Run Open edX Insights
+=====================
+
+#. Log in to the base Open edX Devstack.
```
Comment (dan-f) [inline en_us/install_operations/source/devstack/analytics_devstack.rst]: Log in to the Analytics Devstack.

**Comment by mulby** [comment]
Addressed comments from @dan-f

File: en_us/install_operations/source/front_matter/change_log.rst [hunk 3]
```diff
@@ -10,6 +10,8 @@ Change Log
      - Change
    * - 15 September 2015
      - Added the :ref:`Configuring an edX Instance as an LTI Tool Provider`
```
Comment (lamagnifica) [inline en_us/install_operations/source/front_matter/change_log.rst:12]: I think we lost the word "section." at the end of this line. (thanks for rebasing!)

**Comment by lamagnifica** [comment]
@gabe I built the HTML locally and there were no errors. See my note about a tiny error in the change log, and then merge away. I'll publish as soon as you say go this afternoon.

**Comment by mulby** [comment]
@lamagnifica - merged and ready to publish

**Pull request merged**